service/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# LS docker file
+
+# Tomcat OS base image
+FROM artifacts.bcp.absa.co.za/docker.io/tomcat:9.0.62-jdk8-corretto
+
+LABEL org.opencontainers.image.authors="ABSA OSS"
+LABEL org.opencontainers.image.title="Login service"
+LABEL org.opencontainers.image.base.name="absaoss/login-service:latest"
```

Comment:
At least those 2 can be perhaps parametrized? If this is supposed to be more like company-wide image, not strictly related to LS.

Reply:
Good idea, thanks, will do.

Reply:
Replaced with just `authors=ABSA` in the end.